src/test/java/uk/gov/dhsc/htbhf/steps/TermsAndConditionsSteps.java

```diff
@@ -26,6 +26,13 @@ public void doNotAcceptTermsAndConditions() {
         termsAndConditionsPage.clickContinue();
     }
 
+    @When("^I double the click send application button in terms and conditions page")
+    public void doubleClickSendApplicationButton(){
```

Comment:
Space before {

Reply:
Yes, we don't have Checkstyle or PMD running on the acceptance tests.

Reply:
done